src/main/java/org/tdl/vireo/model/packager/DSpaceSimplePackager.java

```diff
@@ -0,0 +1,102 @@
+package org.tdl.vireo.model.packager;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+//import java.util.zip.ZipEntry;
+//import java.util.zip.FileOutputStream;
+
+import javax.persistence.Entity;
+
+import org.apache.commons.io.FileUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.tdl.vireo.model.FieldValue;
+import org.tdl.vireo.model.Submission;
+import org.tdl.vireo.model.export.DSpaceSimplePackage;
+import org.tdl.vireo.model.formatter.AbstractFormatter;
+import org.tdl.vireo.service.AssetService;
+
+@Entity
+public class DSpaceSimplePackager extends AbstractPackager<DSpaceSimplePackage> {
+
+    public DSpaceSimplePackager() {
+
+    }
+
+    public DSpaceSimplePackager(String name) {
+        setName(name);
+    }
+
+    public DSpaceSimplePackager(String name, AbstractFormatter formatter) {
+        this(name);
+        setFormatter(formatter);
+    }
+
+    // @Override
+    // public DSpaceSimplePackage packageExport(Submission submission, String
+    // manifest){
+    // }
+
+    @Override
+    public DSpaceSimplePackage packageExport(Submission submission, Map<String, String> dsDocs) {
+
+        // Add templated files
+//        List<File> pkg_list = new ArrayList<File>();
+        Map<String,File> pkg_list = new HashMap<String,File>();
+        try {
+            // Add non submitted content
+            for (Map.Entry<String, String> ds_entry : dsDocs.entrySet()) {
+                String docName = ds_entry.getKey();
+                String docContents = ds_entry.getValue();
+                File ff = File.createTempFile(docName,"");
+                FileUtils.writeStringToFile(ff, docContents, "UTF-8");
+                pkg_list.put(docName,ff);
+            }
+            
+
+            /***
```

Comment:
Please remove commented out code or if needed for some reason add documentation explaining when and how it will be used.